pkg/pillar/cmd/diag/diag.go

```diff
@@ -528,22 +528,24 @@ func printOutput(ctx *diagContext) {
 	}
 
 	switch ctx.derivedLedCounter {
-	case 0:
+	case types.LedBlinkUndefined:
 		fmt.Fprintf(outfile, "ERROR: Summary: Unknown LED counter 0\n")
-	case 1:
+	case types.LedBlinkWaitingForIP:
 		fmt.Fprintf(outfile, "ERROR: Summary: Waiting for DHCP IP address(es)\n")
-	case 2:
+	case types.LedBlinkConnectingToController:
 		fmt.Fprintf(outfile, "ERROR: Summary: Trying to connect to EV Controller\n")
-	case 3:
+	case types.LedBlinkConnectedToController:
 		fmt.Fprintf(outfile, "WARNING: Summary: Connected to EV Controller but not onboarded\n")
-	case 4:
+	case types.LedBlinkOnboarded:
 		fmt.Fprintf(outfile, "INFO: Summary: Connected to EV Controller and onboarded\n")
-	case 10:
+	case types.LedBlinkOnboardingFailure:
 		fmt.Fprintf(outfile, "ERROR: Summary: Onboarding failure or conflict\n")
-	case 12:
+	case types.LedBlinkRespWithoutTLS:
 		fmt.Fprintf(outfile, "ERROR: Summary: Response without TLS - ignored\n")
-	case 13:
+	case types.LedBlinkRespWithoutOSCP:
 		fmt.Fprintf(outfile, "ERROR: Summary: Response without OSCP or bad OSCP - ignored\n")
```

Comment:
Can this use the String function output?
You might still need a case to get the ERROR vs INFO vs WARNING but maybe the default case could do all of the errors?

Reply:
Good idea - done.